sql/all-sponsors.sql

```diff
@@ -17,6 +17,9 @@ where
 	((t."OrderId" IS NOT NULL AND t.type LIKE 'CREDIT')
 	OR (t."ExpenseId" IS NOT NULL AND t.type LIKE 'DEBIT'))
 	AND c.type ilike 'organization'
+    AND ((t."RefundTransactionId" IS NOT NULL AND
```

Comment:
Yes I think that should work. @piamancini can try out the query next week on her data set and see if it reflects coinbase properly